tests/test_mock_sampler.py

```diff
@@ -20,13 +20,66 @@
 import dimod.testing as dit
 from dimod import DiscreteQuadraticModel, ExtendedVartype, SampleSet
 
+from dwave.cloud.exceptions import ConfigFileError, SolverNotFoundError
+from dwave.system import DWaveSampler
+import os
```

Comment:
Nit, pep8 [defines an import order](https://www.python.org/dev/peps/pep-0008/#imports) so `import os` should be up at the top with the other built-in packages.